docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -1,72 +1,96 @@
-Configure the Node
-==================
-
-Use the ConfigMap template in ``configuration/config-map.yaml`` file to
-configure the node. Update all the values for the keys in the
-ConfigMaps ``vars``, ``bdb-public-key``, ``bdb-keyring`` and
-``mongodb-whitelist``.
-
-
-Use the Secret template in ``configuration/secret.yaml`` file to configure
-the secrets for this node. Update all the values for the keys in the Secrets
-``mdb-agent-api-key``, ``https-certs``, ``bdb-private-key``,
-``threescale-credentials`` and ``mdb-certs``.
-
-You might not need all the keys during the deployment.
-For example, if you plan to access the BigchainDB API over HTTP, you might
-not need the ``https-certs`` Secret.
-   
-
-Ensure that all the secrets are base64 encoded values and the unused ones
-are set to an empty string.
-For example, assuming that the public key chain is named ``cert.pem`` and
-private key is ``cert.key``, run the following commands to encode the
-certificates into single continuous string that can be embedded in yaml,
-and then copy the contents of ``cert.pem.b64`` in the ``cert.pem`` field,
-and the contents of ``cert.key.b64`` in the ``cert.key`` field. 
-      
+How to Configure Your BigchainDB Node
```

Comment:
Shouldn't this be `How _we_ configure our BigchainDB Node` based on the comment that this is how we set up a cluster and the user can choose any other option available to him/her?